workloads/statefulset/README.md

```diff
@@ -0,0 +1,43 @@
+# StatefulSet Workload
+
+oam-kubernetes-runtime also support K8s native resource, this section will introduce how to use K8s StatefulSet.
+
+## Update RBAC for OAM AppConfig Controller
+
+Make sure your role has below rules:
```

Comment:
How about changing `rules` to `clusterrole` as they are different K8s primitives :)

Reply:
Of course, thanks